Dashboard/va.gov.artemis.ui.data/Models/PatientContact/BreastfeedingCallTab.cs

```diff
@@ -248,6 +256,9 @@ public override string GetNoteText()
                 if (this.BreastfeedingAssessment)
                     sb.AppendLine("Assessed for difficulties with breastfeeding, answered questions, referred to pediatrician as needed");
 
+                if (this.BreastfeedingLactation)
+                    sb.AppendLine("Patient is having difficulties with lactation");
```

Comment:
I am not a fan of the formatting here.  I really want to add braces, but I kept it formatted the same as the rest of the code.  If we were making any other major modifications to the baseline I would vote for changing everything back to proper standards, but for now my vote is to leave it the way we found it.

Reply:
I agree. I'm the same way. I don't like the style with the missing braces. One wrong indentation can mess up the way you may read things.

Reply:
But I agree, we should continue with the same style, until we decide to pull the trigger and change it for the whole code base.